src/nxdoc/nuxeo-server/nuxeo-server-release-notes.md

```diff
@@ -1,1893 +1,609 @@
-- [ELEMENTS-662](https://jira.nuxeo.com/browse/ELEMENTS-662) Two new elements, `<nuxeo-directory-checkbox>` (multiple) and `<nuxeo-directory-radio-group>` (single) added to present and select directory entry(ies).
-- [ELEMENTS-624](https://jira.nuxeo.com/browse/ELEMENTS-624) Exposes `rows` attribute in nuxeo-textarea element.
-- [ELEMENTS-610](https://jira.nuxeo.com/browse/ELEMENTS-610) Replaces video javascript library with browser native video element for to better support browsers.
+We improved some error messages to help analysis and diagnosis.
 
-### Nuxeo AI {{> anchor 'nuxeo-ai'}}
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on the following JIRA tickets:
+- [NXP-28129](https://jira.nuxeo.com/browse/NXP-28129)
+- [NXP-27124](https://jira.nuxeo.com/browse/NXP-27124)
+- [NXP-26528](https://jira.nuxeo.com/browse/NXP-26528)
+- [NXP-27175](https://jira.nuxeo.com/browse/NXP-27175)
+- [NXP-22770](https://jira.nuxeo.com/browse/NXP-22770)
+- [NXP-26687](https://jira.nuxeo.com/browse/NXP-26687)
 
-#### Service to Configure and Serve Ml Custom Models {{since '10.3'}}
+#### javax.mail messages in Nuxeo logs {{Also available in '10.10'}}
 
-Current AI/Machine Learning services for labelling and classification behave mostly in a very generic manner. They do not take into account specific business logic and data, but only broader and more abstract content. As an example, on a picture with cars, they are very able to recognize cars, but for a carmaker, it would be more relevant to detect what would be the car model and year of release. Even more valuable would be maybe to identify the correct car color in the brand's catalogue and detect any car extras.
+`javax.mail` messages are now redirected to Nuxeo logs.
 
-To enable business specific classification and recognition, models need to be build and trained with this information.
-This new service allows customers to build ML custom models that after defined are trained and publish for usage.
-A ML custom model is defined with:
-- NXQL query - to provide the correct dataset of documents to use during training
-- Input fields - the metadata that the model should use to do classifications. Ie. The document image, a set of text fields.
-- Output field - this is the field that the model will try to predict. These could be a field linked to a vocabulary, a document reference or just a `string` metadata.
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-28275](https://jira.nuxeo.com/browse/NXP-28275)
 
-After defined, the custom model has two other main stages:
-- Training - a ML model will be produced by training the repository model with the definition provided
-- Publishing - The result ML model can then be published to be available as service that does Machine Learning classification on the context of a particular business.
+#### nuxeoctl start should be in strict mode by default
```

Comment:
wrong title. Correct one should be "nuxeoctl now starts Nuxeo Platform in strict mode"